pipeline/relation_embedder/batcher/src/main/scala/weco/pipeline/batcher/CLIMain.scala

```diff
@@ -0,0 +1,58 @@
+package weco.pipeline.batcher
+import org.apache.pekko.NotUsed
+import org.apache.pekko.actor.ActorSystem
+import org.apache.pekko.stream.IOResult
+import org.apache.pekko.stream.scaladsl.{
+  Flow,
+  Framing,
+  Sink,
+  Source,
+  StreamConverters
+}
+import org.apache.pekko.util.ByteString
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.Try
+
+object CLIMain extends App {
```

Comment:
Yeah! Perhaps we should end up with this being the default entrypoint to the app when we bin the ECS version so an `sbt "project batcher" run some/input` results in execution routing through here (so we can avoid building a jar between local runs).